mlir/lib/IR/Dialect.cpp

```diff
@@ -31,7 +31,7 @@ static llvm::ManagedStatic<DialectRegistry> dialectRegistry;
 DialectRegistry &mlir::getGlobalDialectRegistry() { return *dialectRegistry; }
 
 // Note: deprecated, will be removed soon.
-static bool isGlobalDialectRegistryEnabledFlag = false;
+static bool isGlobalDialectRegistryEnabledFlag = true; // PlaidML-change Temporary change
```

Comment:
I think we should revert this, let's enable the global registry in the `plaidml` repository.